src/modules/claim/claim.service.ts

```diff
@@ -147,33 +160,82 @@ export class ClaimService {
    * Get claims requested or issued by user with matching DID
    * @param did user DID
    */
-  async getByUserDid(did: string) {
-    return this.claimRepository
+  async getByUserDid({
+    did,
+    filters,
+    user,
+  }: {
+    did: string;
+    filters?: QueryFilters;
+    user?: string;
+  }) {
+    const { isAccepted, parentNamespace } = this.parseFilters(filters);
+
+    const qb = this.claimRepository
       .createQueryBuilder()
       .where(':did = ANY ("claimIssuer")', { did })
-      .orWhere(':did = requester', { did })
-      .getMany();
+      .orWhere(':did = requester', { did });
+
+    if (isAccepted !== undefined) {
+      qb.andWhere('"isAccepted" = :isAccepted', {
+        isAccepted,
+      });
+    }
+
+    if (parentNamespace) {
+      qb.andWhere('"parentNamespace" = :namespace', {
+        namespace: parentNamespace,
+      });
+    }
+
+    if (user) {
+      qb.andWhere(new Brackets(query => {
+        query.where(':user = ANY ("claimIssuer")', {
+          user,
+        }).orWhere(':user = requester', { user })
+      }))
+    }
```

Comment:
What is the difference between `user` and `did`? The logic seems to be duplicated

Reply:
I can change the user to `currentUser`, to make it more clear what is this `user`

Reply:
@dwojno Ok, I think this rename would be a good idea